lib/node_modules/@stdlib/stats/base/dists/pareto-type1/cdf/benchmark/benchmark.js

```diff
@@ -31,22 +31,19 @@ var cdf = require( './../lib' );
 // MAIN //
 
 bench( pkg, function benchmark( b ) {
-	var alpha;
-	var beta;
+	var alpha = uniform( EPS, 100.0 );
+    var beta = uniform( EPS, 100.0 );
```

Comment:
```suggestion
	var alpha;
        var beta;
```

Variable declarations happen at the top and separately from their initializations.